tests/LineScanCameraTests.cpp

```diff
@@ -105,7 +105,7 @@ TEST_F(ConstVelocityLineScanSensorModel, calculateAttitudeCorrection) {
   double attCorr[9];
   adj.resize(15, 0);
   // Pi/2 with simply compensating for member variable m_flyingHeight in UsgsAstroLsSensorModel
-  adj[7] = (M_PI / 2) * 990.0496255790623081338708;
+  adj[7] = (M_PI / 2) * sensorModel->m_flyingHeight;
```

Comment:
Why this change? I guess, why was it hard coded in the first place?

Reply:
This was just hard-coded when the test was written, I think we forgot the sensor model state was public. I just happened to break it, so I made it more flexible.